TKGame/PowerUps/Components/FireStone PowerUps/Related Entities/Shocked.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TKGame.BackEnd;
+using TKGame.Components.Interface;
+using TKGame.Content.Weapons;
+using TKGame.Players;
+using TKGame.Status_Effects;
+
+namespace TKGame.PowerUps.RelatedEntities
+{
+    public class Shocked : Entity
```

Comment:
This looks good but the name "Shocked" for the entity something spawns is a bit confusing with a "Shocked" status as well